manifests/manifest_celery_worker_prod.yml

```diff
@@ -5,10 +5,6 @@ stack: cflinuxfs3
 buildpack: python_buildpack
 env:
```

Comment:
@pkfec can you please add the APP_NAME variable here for prod?

Reply:
good catch @patphongs. I see the variable being added to the file, in my local. Not sure why its not showing up here.

Reply:
my bad, i forgot to save the file. thanks @patphongs . Please check now.

Reply:
All good, thanks!